logstash-plugins/usagetype-plugin/src/test/java/org/reactome/logstash/plugins/usagetype/TestUsageTypePlugin.java

```diff
@@ -0,0 +1,108 @@
+package org.reactome.logstash.plugins.usagetype;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNull;
+
+import java.util.Collection;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.concurrent.atomic.AtomicInteger;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.logstash.plugins.ConfigurationImpl;
+import org.logstash.plugins.ContextImpl;
+import org.reactome.logstash.plugins.usagetype.util.IPRangeTree;
+
+import co.elastic.logstash.api.Configuration;
+import co.elastic.logstash.api.Context;
+import co.elastic.logstash.api.Event;
+import co.elastic.logstash.api.FilterMatchListener;
+
+public class TestUsageTypePlugin
+{
+
+	private Configuration config;
+	private Context context;
+	private UsageType filter;
```

Comment:
Can this be given a different name?  Since the `filter` method gets called on this object, it is a bit confusing to see `filter.filter(...)`

Reply:
In the context of Logstash, it _is_ a filter.
```Java
public class UsageType implements Filter
{
...
```